bears/general/FilenameBear.py

```diff
@@ -63,6 +71,8 @@ def run(self, filename, file,
                           'Using the default "snake" naming convention.')
                 file_naming_convention = 'snake'
 
+        messages = []
```

Comment:
Nope this is on purpose, so we can generate a single patch without conflict errors :+1: